src/pages/orders/components/OrderDetailsCard/OrderDetailsCardHeader/OrderDetailsCardHeader.tsx

```diff
@@ -11,11 +11,10 @@ const OrderDetailsCardHeader = () => {
     const {
         displayPaymentAmount,
         hasTimerExpired,
-        id,
         isBuyerConfirmedOrder,
         isPendingOrder,
-        local_currency: localCurrency,
         orderExpiryMilliseconds,
+        p2p_order_info,
```

Comment:
```suggestion
        p2p_order_info: p2pOrderInfo,
```

haven't enforced the camel casing rules since hooks were using snake cases.. but we can ensure we use only camelcase for components